internal/config/config.go

```diff
@@ -33,6 +33,11 @@ type Config struct {
 	Database database.DBConf
 }
 
+func init() {
+	// Convert logs to JSON format in all the services
+	log.SetFormatter(&log.JSONFormatter{})
+}
+
```

Comment:
Place this in the `NewConfig` block.
It would also be nice to make it configurable so that the users can switch at their own pace since this is directly related to how logging is stored and parsed.